presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -261,6 +263,9 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
         String jdbcTypeName = typeHandle.getJdbcTypeName()
                 .orElseThrow(() -> new PrestoException(JDBC_ERROR, "Type name is missing: " + typeHandle));
 
+        if (jdbcTypesMappedToVarchar.contains(jdbcTypeName)) {
```

Comment:
This needs to be added to MySqlClient and SqlServerClient as well, since they too override `toPrestoType()`